<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_DocumentPreamble</class>
 <widget class="QDialog" name="Dialog_DocumentPreamble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>313</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>682</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>313</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaTeX Office Writer</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>661</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDocumentBeginning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Document Beginning</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="4" column="0">
     <widget class="QLabel" name="labelDocumentAbstract">
      <property name="text">
       <string>Document Abstract</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDocumentTitle"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelDocumentAuthor">
      <property name="text">
       <string>Document Author</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelDocumentTitle">
      <property name="text">
       <string>Document Title</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDocumentDate"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDocumentDate">
      <property name="text">
       <string>Document Date</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButtonEnableAbstract">
      <property name="text">
       <string>Enable abstract</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QRadioButton" name="radioButtonDisableAbstract">
      <property name="text">
       <string>Disable abstract</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDocumentAuthor"/>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="plainTextEditDocumentAbstract">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDocumentTitle</tabstop>
  <tabstop>lineEditDocumentAuthor</tabstop>
  <tabstop>lineEditDocumentDate</tabstop>
  <tabstop>radioButtonEnableAbstract</tabstop>
  <tabstop>radioButtonDisableAbstract</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_DocumentPreamble</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_DocumentPreamble</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
